soccer/gameplay/robocup-py.cpp

```diff
@@ -246,6 +246,10 @@ void State_draw_polygon(SystemState *self, boost::python::list points, boost::py
 	self->drawPolygon(ptVec, Color_from_tuple(rgb), QString::fromStdString(layer));
 }
 
+void State_draw_raw_polygon(SystemState *self, Geometry2d::Polygon points, boost::python::tuple rgb, const std::string &layer) {
```

Comment:
If we make `points` a pointer, it'll reduce the overhead of having to make a copy.